docs/config/README.md

```diff
@@ -1760,6 +1761,42 @@ The module will be shown if any of the following conditions are met:
 format = "via [🐪 $version]($style) "
 ```
 
+## OpenStack
+
+The `openstack` module shows the current OpenStack cloud and project . This is based on
```

Comment:
Nitpick, but there is an extra space here.

```suggestion
The `openstack` module shows the current OpenStack cloud and project. This is based on
```

Reply:
Fixed on my side. Will push later.